src/File/Hash.php

```diff
@@ -163,7 +163,7 @@ public function isValid($value, $file = null)
             }
 
             foreach ($hashes as $hash) {
-                if ($filehash === $hash) {
+                if ($filehash == $hash) {
```

Comment:
I think we should leave it as it is and in the other place assure that `$hash` is always string.